micro-elasticache/build.gradle

```diff
@@ -0,0 +1,89 @@
+apply plugin: 'groovy'
+apply plugin: 'java'
+repositories {
+	
+	maven { url "https://jitpack.io" }
+}
+
+description = 'micro-elasticache'
```

Comment:
elasticache, perhaps I should add a hyphen to make it elasti-cache? https://console.aws.amazon.com/elasticache